plugins/modules/aws_secret.py

```diff
@@ -166,8 +172,18 @@ def create_args(self):
         }
         if self.description:
             args["Description"] = self.description
+        if self.replica_regions:
+            args["AddReplicaRegions"] = self.replica_regions
         if self.kms_key_id:
             args["KmsKeyId"] = self.kms_key_id
+        if self.replica_regions:
+            add_replica_regions = []
+            for region in self.replica_regions:
+                if self.kms_key_id:
+                    add_replica_regions.append({'Region': region, 'KmsKeyID': self.kms_key_id})
```

Comment:
what happens if the kms key id is not available in the replicated region?  
imo kms are region based and not global. do we need a `replica_region_kms_key_id`?

Reply:
You are correct, we will need a new variable.  I initially thought that this was to specify the kms key id used in the source region.